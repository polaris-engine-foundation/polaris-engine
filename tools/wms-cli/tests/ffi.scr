func main() {
    array["hello"] = "hello";
    a = hello(1, 3.14, "str", array);
    print(a["hello"]);
}
