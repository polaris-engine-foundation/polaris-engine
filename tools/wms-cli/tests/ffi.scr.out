In FFI hello(): got a = 1
In FFI hello(): got b = 3.140000
In FFI hello(): got c = str
In FFI hello(): got d["hello"] = hello
bonjour
